evennia/accounts/migrations/0001_initial.py

```diff
@@ -8,7 +8,7 @@
 
 class Migration(migrations.Migration):
 
-    dependencies = [("auth", "0001_initial"), ("typeclasses", "0001_initial")]
+    dependencies = [("auth", "0001_initial"), ("typeclasses", "0001_initial"), ("contenttypes", "__first__")]
```

Comment:
Hm, these definitely don't belong with this PR ...

Reply:
truth. they probably dont even belong in my directory.